test/powershell/engine/Basic/DefaultCommands.Tests.ps1

```diff
@@ -240,6 +241,7 @@ Describe "Verify aliases and cmdlets" -Tags "CI" {
 "Cmdlet",       "ConvertFrom-StringData",           "",                                 $($FullCLR -or $CoreWindows -or $CoreUnix),     "",                     "",                     "None"
 "Cmdlet",       "Convert-Path",                     "",                                 $($FullCLR -or $CoreWindows -or $CoreUnix),     "",                     "",                     "None"
 "Cmdlet",       "Convert-String",                   "",                                 $($FullCLR                               ),     "",                     "",                     ""
+"Cmdlet",       "ConvertTo-CliXml",                 "",                                 $($FullCLR -or $CoreWindows -or $CoreUnix),     "",                     "",                     "None"
```

Comment:
I wonder if the cmdlets were in Windows PowerShell.

Reply:
I dont believe so. They are new for all OS platforms.

Reply:
I mean we should remove `$FullCLR`

Reply:
Fixed in 0a72c4aee